src/Microsoft.Win32.SystemEvents/Directory.Build.props

```diff
@@ -4,5 +4,6 @@
   <PropertyGroup>
     <AssemblyVersion>4.0.1.0</AssemblyVersion>
     <AssemblyKey>Open</AssemblyKey>
+    <IsNETCoreApp>true</IsNETCoreApp>
```

Comment:
Looks like you got everything except you didn't remove the dependency in Microsoft.Win32.SystemEvents on System.ComponentModel.TypeConverter. Please remove that reference in the project file.

Reply:
Missed it. Thanks.

I wonder if tests will pass. On my computer `System.ComponentModel.TypeConverter` tests are failing randomly with error code -1073741795. I pushed it anyway to see the results here.